server/views/index.hbs

```diff
@@ -8,11 +8,9 @@
     </tr>
   </thead>
   <tbody>
-    {{#each articles}}
+    {{#each articles}}{{#if this.visible}}
```

Comment:
@kharazzoro .. `views/index.hbs` and `admin-list-articles` both have a list of articles and the code is duplicated. Could you DRY it -maybe move the list code to a partial and use it in both views

Reply:
Do I need to use partials, and is does it work int hbs same in handalebars